### DISCUSSION TITLE
[Feature] Share all photos associated with a given Person as an album

### DISCUSSION THREAD

**Discussion by lpryszcz** [immich-app/immich#6877] [Feature Request]
### The feature

Could all photos associated with a given Person (from People) be shared as an album? Similarly to the other service? 
New photos recognised as a given Person, would need to be automatically added to that Person album.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #6755
